test/unit/org/apache/cassandra/db/commitlog/CommitLogSegmentManagerCDCTest.java

```diff
@@ -327,7 +326,10 @@ public void testReplayLogic() throws Throwable
             for (CDCIndexData cid : oldData)
             {
                 if (cid.fileName.equals(ncid.fileName))
+                {
                     found = true;
+                    break;
```

Comment:
Nice, we can add an identical `break` in the similar exists loop a few lines above (line 308)

Reply:
Done